pom.xml

```diff
@@ -46,7 +46,8 @@
         <version.kafka>3.2.1</version.kafka>
         <version.org.slf4j>1.7.36</version.org.slf4j>
         <version.logback>1.4.0</version.logback>
-        <version.ybclient>0.8.32-20221024.104302-1</version.ybclient>
+        <!-- <version.ybclient>0.8.32-20221024.104302-1</version.ybclient> -->
+        <version.ybclient>0.8.33-SNAPSHOT</version.ybclient>
```

Comment:
Temporarily changing the yb-client version for a build to be done locally since no jar is available on S3 yet.